Sources/SourceKitBazelBSP/RequestHandlers/SKOptions/CompilerArgumentsProcessor.swift

```diff
@@ -141,6 +75,15 @@ enum CompilerArgumentsProcessor {
 
         var compilerArguments: [String] = []
 
+        // drop clang and swiftc compiler call
+        // FIXME: handle other language types when adding other language support
+        var rawArguments = rawArguments
+        switch language {
+        case .swift: rawArguments.removeFirst(2)
+        case .objective_c: rawArguments.removeFirst()
+        default: break
+        }
```

Comment:
For better performance, we can instead just set the index to skip those (need to move the index var to be above this too)

```
        switch language {
        case .swift: index = 2
        case .objective_c: index = 1
        default: break
        }
        ```